package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "nodemon",
     "start": "probot run ./index.js",
     "test": "npm-run-all --print-label --parallel lint:* --parallel test:*",
-    "lint:js": "standard",
+    "lint:js": "prettier-standard --format --lint",
```

Comment:
i assume this cli tool is available through some tool that we already depend on since there is no added dependency as part of this. i usually find it safer to declare tools that we use directly as direct (dev) dependencies. could this be added?

Reply:
I missed that!

Reply:
Fix in ac4f30f27688cad37a486eb75685e7534ffbf719